### ISSUE TITLE
Error with ssl

### ISSUE THREAD

**Issue by Sahar120504** [Dpeta/pesterchum-alt-servers#167]
when i run Pesterchum it says "SSL: WRONG VERSION NUMBER". Sorry for bothering
![изображение_2024-02-06_204531759](https://github.com/Dpeta/pesterchum-alt-servers/assets/159177929/509fa6cc-6bdb-4f96-b38c-3055f38657d9)

**Comment by Dpeta** [comment]
Wrong ssl version number tends to get thrown for a bunch of misc. reasons, so not sure why that'd be exactly. Could always try the pre-release maybe? https://github.com/Dpeta/pesterchum-alt-servers/releases/tag/v2.6.0rc1

Otherwise maybe it's some kinda network issue?

**Comment by Dpeta** [comment]
Yeah still not sure why it'd do that honestly. Maybe your firewall doesn't allow IRC connections or smthin?
Some stuff you could try:

 - Connecting without SSL/TLS, you can add a new server as ``irc.pesterchum.xyz:6667`` without the SSL/TLS option checked. If it's a firewall issue it still won't connect but might give a more descriptive error.
   - be mindful of the fact that this does mean messages will be send in plaintext. 
 - Check if [pesterchum godot](https://gitlab.com/mocchapi/pesterchum-godot#getting-started) or [pco](https://pesterchum.online/) does work. Because PCO uses a websocket connection it probably won't be blocked by firewalls that don't allow IRC.
 - Connecting over a [VPN](https://www.privacyguides.org/en/vpn/), you can ask for a whitelist in the support server if it's dronebl listed. If it's a firewall thing that'll probably get around it.
